### PR TITLE
opera target view

### PR DESCRIPTION
**Pull request by Haishi2016**
This should be safe to merge as it affects only the opera portal.

### CODE REVIEW

**Comment by msftcoderdjw** [comment]
Hi @Haishi2016 , eclipsefdn/eca checke failed in this PR. Can you please take a look?

File: README.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 
 _(last edit: 02/02/2024)_
 
+### ⚠️⚠️⚠️ This branch hosts experimental features. The Symphony community does not guarantee long-term support for these features. Their incorporation into the main branch is not assured, and they may be deprecated or removed at any time without notice. For more details, please see [Experimental Feature Table](./experimental-features.md)⚠️⚠️⚠️
```
Comment (msftcoderdjw) [inline README.md:8]: Not include this in main.

File: api/pkg/apis/v1alpha1/managers/stage/stage-manager.go [hunk 2]
```diff
@@ -232,6 +239,7 @@ func (s *StageManager) ResumeStage(status model.ActivationStatus, cam model.Camp
 						TriggeringStage:      stage,
 						Schedule:             cam.Stages[nextStage].Schedule,
 						Namespace:            namespace,
+						Proxy:                cam.Stages[nextStage].Proxy,
```
Comment (msftcoderdjw) [inline api/pkg/apis/v1alpha1/managers/stage/stage-manager.go:242]: These are the proxy processor changes, please remove them.

**Review by msftcoderdjw — CHANGES_REQUESTED** [review]

**Comment by Haishi2016** [comment]
> Hi @Haishi2016 , eclipsefdn/eca checke failed in this PR. Can you please take a look?

This is because there's a commit from action@github.com ....

File: api/pkg/apis/v1alpha1/managers/stage/stage-manager.go [hunk 3]
```diff
@@ -50,6 +50,13 @@ func (t *TaskResult) GetError() error {
 		switch sv := v.(type) {
 		case v1alpha2.State:
 			break
+		case float64:
```
Comment (msftcoderdjw) [inline api/pkg/apis/v1alpha1/managers/stage/stage-manager.go:53]: Just curious in which case, we will get float64 status?

File: docs/samples/opera/app/types.d.ts [hunk 4]
```diff
@@ -77,6 +77,79 @@ export interface CatalogState {
...
+}
+
+export interface ObjectMeta {
+    namespace: string;
+    name: string;
+    labels: Record<string, string>;
+    annotations: Record<string, string>;
+}
+
+export interface TargetState {
```
Comment (msftcoderdjw) [inline docs/samples/opera/app/types.d.ts:147]: I saw in this file, there's still some object like SolutionState which doesn't align new object model. Do we need to fix them?

**Pull request closed without merging**